app/workers/evss/disability_compensation_form/evss_document.rb

```diff
@@ -3,6 +3,7 @@
 require 'central_mail/datestamp_pdf'
 require 'pdf_fill/filler'
 
+# [wipn8923] verify that this is replaced by LH object
```

Comment:
This appears to only be used as the superclass for https://github.com/department-of-veterans-affairs/vets-api/blob/dbex-64564-lighthouse-supporting-evidence-attachment/app/workers/evss/disability_compensation_form/form8940_document.rb, which I believe is the supplemental PTSD form.  There is some code in here that is duplicated elsewhere unnecessarily so I don't think we'll copy everything. It does appear we will need to build a Lighthouse equivalent of some of this to port that form's behavior